src/modules/grants/components/GrantSubscriptionSection.tsx

```diff
@@ -1,235 +1,58 @@
+import { Body, H3 } from '@/shared/components/typography'
+import { getPath } from '@/shared/constants'
 
-const GRANT_PROJECT_ID = __development__ ? '6' : '123'
+const AIRTABLE_FORM_URL = 'https://airtable.com/appyM7XlNIWVypuP5/tbll81HvN1CY7GC1x/viwbkXQNuMPFscFBr?blocks=hide'
 
-type GrantGoal = Pick<ProjectGoal, 'id' | 'title' | 'progress' | 'description' | 'amountContributed' | 'currency'>
-
-interface GrantGoalCardProps {
-  goal: GrantGoal
-  onSubscribe: () => void
-}
-
-const GoalProgressBar = ({
-  percentage,
-}: Pick<BoxProps, 'width' | 'bg'> & {
-  percentage: number
-}) => {
-  const isMobile = useMobileMode()
-
-  const displayPercentage =
-    percentage === 0 ? '0%' : isMobile ? `${Math.round(percentage)}%` : `${percentage.toFixed(0)}%`
-
-  return (
-    <HStack width="100%" height="24px" justifyContent="flex-start" borderRadius="44px" bg="neutral1.3">
-      <HStack
-        p={'5px'}
-        width={`${percentage}%`}
-        minWidth="30px"
-        height="24px"
-        bgColor={'neutral1.11'}
-        borderRadius="44px"
-        justifyContent={'flex-end'}
-        alignItems="center"
-      >
-        <Body size="xs" medium color={'neutral1.1'}>
-          {displayPercentage}
-        </Body>
-      </HStack>
-    </HStack>
-  )
-}
-
-const GrantGoalCard: React.FC<GrantGoalCardProps> = ({ goal, onSubscribe }) => {
-  const { formatUsdAmount, formatSatsAmount, formatAmount } = useCurrencyFormatter()
-
-  const amountContributed = formatAmount(goal.amountContributed, goal.currency)
-  const usdAmount = formatUsdAmount(goal.amountContributed)
-  const satsAmount = formatSatsAmount(goal.amountContributed)
+/** Section prompting users to subscribe to upcoming grant notifications or make a one-off contribution */
+export const GrantSubscriptionSection = () => {
+  const navigate = useNavigate()
 
   return (
-    <CardLayout p={6} height="100%" display="flex">
-      <VStack align="stretch" spacing={4} flex={1}>
-        <H2 size="xl">{goal.title}</H2>
-        <Body color="neutral1.11" flex={1}>
-          {goal.description}
-        </Body>
-        <VStack spacing={4} align="stretch">
-          <GoalProgressBar width="100%" percentage={goal.progress} />
-          <Body color="neutral1.11">
-            {amountContributed} ({goal.currency === ProjectGoalCurrency.Btcsat ? `${usdAmount}` : `${satsAmount}`})
-          </Body>
-          <Button onClick={onSubscribe} size="lg" variant="solid" colorScheme="primary1">
-            {t('$5 / month')}
-          </Button>
+    <CardLayout p={6}>
+      <Flex direction={{ base: 'column', md: 'row' }} align={{ base: 'stretch', md: 'center' }} gap={6}>
+        <VStack align="start" spacing={2} flex={1}>
+          <VStack align="start" spacing={1} w="90%">
+            <H3 size="lg" bold>
+              {t('Support Geyser Grants')}
+            </H3>
+            <Body size="md">{t('Help fund the next round — or get an email when it launches.')}</Body>
```

Comment:
You can add this to en.json, or run the make language command in the development repo, so that these are auto added when you develop.